API.md

```diff
@@ -531,10 +534,31 @@ Assert that the `value` property of an [HTMLInputElement][] is empty.
 
 #### Parameters
 
--   `message` **[string](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String)?** 
+-   `message` **[string](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String)?**
```

Comment:
where are these changes coming from? did you generate the file or manually adjust it in the editor?